include/graphblas/utils/iscomplex.hpp

```diff
@@ -74,6 +74,13 @@ namespace grb {
 					return x;
 				}
 
+				/**
+				 * @returns The absolute value squared of a given value.
+				 */
+				static C norm( const C &x ) noexcept {
+					return x;
```

Comment:
should this not call `fabs` or do a comparison for negative and then negate?

Reply:
Also, are there any assumptions on the type `C` that could be static-asserted? (E.g., is C assumed to be floating-point?)